sdk/spring/AZURE_SPRING_BOMS_USAGE.md

```diff
@@ -0,0 +1,43 @@
+# How to use azure spring BOMs
+
+### Adding the azure-spring-boot-bom BOM
+
+To consume the azure spring boot BOM include it in the `dependencyManagement` section of your project's POM. 
+Note that this **does not** result in all dependencies being included in your project.
+
+[//]: # ({x-version-update-start;com.azure.spring:azure-spring-boot-bom;dependency})
```

Comment:
@moarychan did you try to run the update version script locally to see whether this will be upgraded when we bump up the bom version?

Reply:
I checked that as long as it can be scanned by the **update_versions.py** script, it will be updated, and it must be configured in version_client.txt.

Reply:
Does this mean we should add these boms to version_client.txt in this PR?

Reply:
I think it is, but currently, the azure-spring-boot-bom is also not added.

Reply:
The version tag azure-spring-boot-bom has been added in version_client.txt.